nylas/client/restful_models.py

```diff
@@ -638,6 +645,7 @@ class Event(NylasAPIObject):
         "recurrence",
         "status",
         "master_event_id",
+        "job_status_id",
```

Comment:
this job status doesn't always get returned from /event, will this parsing be lenient if it None?

Reply:
@jseller That is correct, parsing will be lenient if any of the values are `None`